src/OpenSage.Game/Game.cs

```diff
@@ -402,12 +403,7 @@ public NetworkMessageBuffer NetworkMessageBuffer
                     _userDataFileSystem = AddDisposable(new FileSystem(FileSystem.NormalizeFilePath(UserDataFolder)));
                     ContentManager.UserDataFileSystem = _userDataFileSystem;
 
-                    // TODO: Re-generate MapCache.ini for user maps
-                    var file = _userDataFileSystem.GetFile(@"Maps\MapCache.ini");
-                    if (file != null)
-                    {
-                        ContentManager.LoadIniFile(file);
-                    }
+                    new UserMapCache(ContentManager).Initialize(AssetStore);
```

Comment:
Is it on purpose that you don't store the instance? If so, shouldn't this be static?

Reply:
It's not static because it stores the `ContentManager` in a field and uses it in multiple methods...but other than that, it's work is done after `Initialize`. Maybe `UserMapCacheInitializer` would be a better name?

Reply:
I think we can leave it as is if there's no other use for that instance. I was just confused since it's a bit unusual to see something like this :)